src/util/util.c

```diff
@@ -589,6 +589,14 @@ errno_t sss_filter_sanitize_ex(TALLOC_CTX *mem_ctx,
             output[j++] = '5';
             output[j++] = 'c';
             break;
+        case '\n':
+            output[j++] = '\\';
+            output[j++] = '0';
+            output[j++] = 'd';
+            output[j++] = '\\';
```

Comment:
My fault, I went to the CR+LF without a second thought; 0x0a should be enough.

Regarding \r, I haven't tested it, but I don't think it'll be required.